### ISSUE TITLE
Feature request: add "glmmTMB" objects.

### ISSUE THREAD

**Issue by skanskan** [leifeld/texreg#130]
Please add the option to accept glmmTMB summaries.
They are similar to lme4 but has also the AIC information.
I would like to convert glmmTMB summary to Latex. It's a very fast library to perform mixed effect models.

This would be a simple example:
```
Family: binomial  ( logit )
Formula:          Outcome ~ var1 + var2  (1 | ID)
Data: mydata

     AIC      BIC   logLik deviance df.resid 
 1183412  1181516  -591103  1183106   985618 

Random effects:

Conditional model:
 Groups Name        Variance Std.Dev.
ID    (Intercept) 0.3338   0.5777  
Number of obs: 1000000, groups:  ID, 55

Conditional model:
             Estimate Std. Error z value Pr(>|z|)    
(Intercept)  0.334715   0.032700    2.77   0.0569 .  
Outcome       -0.066648   0.004529  -14.72   <2e-16 ***
var1                0.211224   0.003106   68.00   <2e-16 ***
var2              -0.271511   0.001299 -208.95   <2e-16 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

```

**Comment by leifeld** [comment]
Thanks. We will get to this after clearing some of the other existing issues. If you would like to speed it up, please feel free to provide an `extract` method either here as a comment or as a pull request. There is a tutorial [here](http://stackoverflow.com/questions/38894044/print-beautiful-tables-for-h2o-models-in-r/39135080#39135080) on how to write these extensions.

**Comment by leifeld** [comment]
This is being implemented in pull request #144 and should be included very soon.
